### PR TITLE
refactor(VAlert): tuning pass

### PR DESCRIPTION
**Pull request by johnleider**
## Release Review Checklist
The following items must be completed before merging.

* [x] Code styling / Coding guidelines
* [x] Unit and E2E tests
* [x] Documentation

## Markup:

<details>

```vue
<template>
  <v-app>
    <v-main>
      <div class="ma-4 pa-4">
        <v-alert
          title="Alert"
          text="Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto illum architecto delectus voluptas cumque quia? Magnam dolore neque ratione veritatis? Fuga voluptas magni incidunt cumque doloremque itaque iste, in velit."
          closable
          icon="mdi-vuetify"
        />

        <br>

        <v-alert>
          <template #prepend>
            <v-icon>mdi-vuetify</v-icon>
          </template>

          <template #title>
            <span>Alert</span>
          </template>

          <template #append>
            Append slot
          </template>

          <template #text>
            <div>
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto illum architecto delectus voluptas cumque quia? Magnam dolore neque ratione veritatis? Fuga voluptas magni incidunt cumque doloremque itaque iste, in velit.
            </div>
          </template>
        </v-alert>

        <br>

        <v-alert>
          <v-alert-icon>mdi-vuetify</v-alert-icon>

          <v-alert-title>Alert</v-alert-title>

          <v-alert-text>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto illum architecto delectus voluptas cumque quia? Magnam dolore neque ratione veritatis? Fuga voluptas magni incidunt cumque doloremque itaque iste, in velit.
          </v-alert-text>

          <template #close>
            <v-icon />
          </template>
        </v-alert>
      </div>
    </v-main>
  </v-app>
</template>

<script setup>
  // import { computed, ref } from 'vue'
</script>
```
</details>

### CODE REVIEW

File: packages/vuetify/src/components/VAlert/VAlert.sass [hunk 1]
```diff
@@ -1,12 +1,14 @@
 @forward './variables'
+
+// Imports
```
Comment (KaelWD) [inline packages/vuetify/src/components/VAlert/VAlert.sass:3]: No shit sherlock

File: packages/docs/src/pages/en/components/alerts.md [hunk 2]
```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
...
+<example file="v-alert/prop-density" />
 
-The **colored-border** prop removes the alert background in order to accent the **border** prop. If a **type** is set, it will use the type's default color. If no **color** or **type** is set, the color will default to the inverted color of the applied theme (black for light and white/gray for dark).
+The **density** prop supports 3 levels of component height; **default**, **comfortable**, and **compact**.
 
-<example file="v-alert/prop-colored-border" />
+#### Variants
 
-#### Density
+`v-alert` has 6 style variants, **contained, contained-flat, contained-text, outlined, plain,** and **text**. By default, the `v-alert` component is **contained-flat**; which means that it has a solid background and does not have a box-shadow (elevation).
```
Comment (KaelWD) [inline packages/docs/src/pages/en/components/alerts.md:85]: I don't think we need this on every single component page that supports variants

File: packages/docs/src/pages/en/components/alerts.md [hunk 3]
```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
...
 
-The **density** prop decreases the height of the alert based upon 1 of 3 levels of density; **default**, **comfortable**, and **compact**.
+<example file="v-alert/prop-variant" />
 
-<example file="v-alert/prop-density" />
+All Vuetify components are globally configurable which makes it easy to create your own design system using our framework. The following example demonstrates how to set the default **variant** and other properties globally for `v-alert`:
```
Comment (KaelWD) [inline packages/docs/src/pages/en/components/alerts.md:89]: And we definitely don't need this on every component page

File: packages/docs/src/pages/en/components/alerts.md [hunk 4]
```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
...
+Unlike the `v-banner` component, `v-alert` is intended to be re-used throughout your application as call-outs that draw a user's attention.
 
 ### Props
 
+In addition to the standard [v-sheet](/components/sheets/) properties such as elevation, dimension, and rounded, `v-alert` supports **v-model**, [variants](), and [density](); and many more.
+
 #### Type
 
-The **type** prop provides 4 default `v-alert` styles: **success**, **info**, **warning**, and **error**. Each of these styles provides a default icon and color. The default colors can be configured globally by customizing [Vuetify's theme](/features/theme).
+The `v-alert` component has 4 contextual states: **success**, **info**, **warning**, and **error**. Each state has a default _color_ and _icon_ associated with it. The following example uses the **type** prop to tell the `v-alert` that it's in an **error** state:
```
Comment (KaelWD) [inline packages/docs/src/pages/en/components/alerts.md:65]: You use bold/italic text a lot, is there some logic to when you use those vs ``` `` ```?

File: packages/docs/src/pages/en/components/alerts.md [hunk 5]
```diff
@@ -98,25 +153,49 @@ The **outlined** prop inverts the style of an alert, inheriting the currently ap
...
+| $alert-border-opacity | `.38` | The default opacity used for borders |
+| $alert-border-padding | `16px` | The default padding applied to the root `v-alert` element |
+| $alert-border-plain-opacity | `.62` | The dimmed opacity used with the **plain** variant |
+| $alert-border-plain-transition | `.2s opacity $settings.$standard-easing` | Standard transition |
+| $alert-border-positions | `absolute fixed sticky` | Supported through the **position** prop, i.e. **position="fixed"** |
+| $alert-border-prepend-margin-inline-end | `16px` | The spacing between the prepend slot and main content area |
+| $alert-border-radius | `settings.$border-radius-root` | Uses the root border-radius from global variables |
+| $alert-border-style | `settings.$border-style-root` | Uses the root border-style from global variables |
+| $alert-border-thin-width | `8px` | The value used when applying a border |
+| $alert-border-width | `0` | The default value used for borders |
```
Comment (KaelWD) [inline packages/docs/src/pages/en/components/alerts.md:192]: These were auto-generated in v2

File: packages/docs/src/pages/en/components/alerts.md [hunk 6]
```diff
@@ -52,40 +52,95 @@ The recommended placement of elements inside of `v-alert` is:
 | [v-alert](/api/v-alert/) | Primary Component |
 | **v-alert-title** | Functional Component used to display the `v-alert` title. Wraps the `#title` slot |
 
-## Examples
+## Implementation
+
+Unlike the `v-banner` component, `v-alert` is intended to be re-used throughout your application as call-outs that draw a user's attention.
```
Comment (KaelWD) [inline packages/docs/src/pages/en/components/alerts.md:57]: callout isn't hyphenated

File: packages/docs/src/pages/en/components/alerts.md (hunk 4 above)
Comment (johnleider) [reply packages/docs/src/pages/en/components/alerts.md:65]: I typically only use ` for components or code related emphasis.

**Pull request merged**